script.js

```diff
@@ -191,21 +197,17 @@ const onMapLoad = async () => {
           seekingVolunteers: item.gsx$seekingvolunteers.$t,
           seekingMoney: seekingMoney,
           urgentNeed: item.gsx$urgentneed.$t,
-          notes: item.gsx$notes.$t,
-          mostRecentlyUpdatedAt: item.gsx$mostrecentlyupdated.$t
+          notes: item.gsx$notes.$t
         }
         const location = _.pickBy(rawLocation, val => val != '')
         const status = getStatus(item.gsx$color.$t)
 
-        if (!status) {
-          throw new Error("Malformed data for " + location.name + ", could not find status: " + item.gsx$color.$t)
-        }
-
```

Comment:
Curious why you're removing this? I gets caught and outputted to the log, which I think can be helpful so we know if the data sheet is broken.

Reply:
I started getting a console error due to some funky data on the `Not a thing` row, pretty sure it's just missing the status color/id. Adding in the default to unknown status was my workaround for it in https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/pull/79/commits/5fefd9bd0ebcabd257e0ca0e99c57e74ce771c4a. 

With this error in, as JS error was being raised and this QA item wasn't passing:
`Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.`

![image](https://user-images.githubusercontent.com/852709/83885736-638f4000-a70c-11ea-9195-877471a820d9.png)

(slack link: https://opentwincities.slack.com/archives/C01525CPU5A/p1591330752048200)